app/views/observations/show/_name_info.html.erb

```diff
@@ -1,22 +1,24 @@
 <% # Section showing information about consensus name in ShowObservation
-  lines = []
-  name = observation.name
+lines = []
+name = obs.name
 %>
 
-<%= content_tag(:p,
-                link_to(:show_name.t(name: name.display_name_brief_authors),
-                        name_path(name.id))) %>
-<%= content_tag(:p, link_to("MyCoPortal", mycoportal_url(name),
-                            target: :_blank)) %>
-<%= content_tag(:p, link_to("Mycobank", mycobank_name_search_url(name),
-                            target: :_blank)) %>
-<%= content_tag(:p, link_to(:show_observation_more_like_this.t,
-                            observations_path(name: name.id))) %>
-<%= content_tag(:p, link_to(:show_observation_look_alikes.t,
-                            observations_path(name: name.id,
-                                              look_alikes: "1"))) %>
-<%= content_tag(:p, link_to(:show_observation_related_taxa.t,
-                            observations_path(name: name.id,
-                                              related_taxa: "1"))) %>
-<%= list_descriptions(object: name)&.map { |link|
-      content_tag(:div, link) }.safe_join %>
+<%=
+panel_with_outer_heading(
+  id: "observation_name_info",
+  heading: :show_name.t(name: name.display_name_brief_authors),
+  inner_class: "name-section small"
+) do %>
+
+  <%= tag.div(class: "row") do %>
+    <%= tag.div(class: "col-xs-6") do
+      concat(tag.div("On MO:", class: "font-weight-bold"))
```

Comment:
These should probably be translated strings.

Reply:
good catch, thanks! i mocked those up on BS branch and forgot.

What do you think about the box title being "About this taxon", rather than "About {name}"?
Wondering because the "About {name}" wording is repeated for the MO name link below.

Reply:
Added the translation strings. 

Here it is with "About this taxon" in case @JoeCohen  wants to weigh in.

<img width="497" alt="Screen Shot 2023-08-23 at 3 40 29 PM" src="https://github.com/MushroomObserver/mushroom-observer/assets/1948095/802427a1-9840-46bd-92a1-7f786e227398">